content/docs/user-guide/gems/reference/ros2/ros2-gem-user-guide.md

```diff
@@ -0,0 +1,209 @@
+---
+linkTitle: Guide
+title: O3DE ROS 2 Gem Details
+description: User guide to develop ROS 2 enabled simulation
+toc: true
+---
+
+The O3DE ROS 2 Gem and its build instructions can be found in [this repository](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2). The ROS 2 Gem helps to build robotic simulations with [ROS 2 / Robot Operating System](https://www.ros.org/).An example of usage can be seen at [Robot Vacuum Sample](https://github.com/o3de/RobotVacuumSample).
```

Comment:
At the start of a page, it's important to have a few sentences introducing the purpose of the page. In addition to the background information you provide, I highly suggest defining "what information can the user find in this page".

Reply:
Added a few sentences here with explanations.